hnn_core/network.py

```diff
@@ -258,6 +258,64 @@ def _add_external_feeds(self):
         # Create the feed dynamics (event_times)
         self._instantiate_feeds(n_trials=self.params['N_trials'])
 
+    def add_poisson_drive(self, name, t0, T, lamtha, target_cell_type,
+                          weight_ampa, weight_nmda, spatial_lamtha, loc, prng):
+        """Add poisson drive target a specific cell type
+
+        Parameters
+        ----------
+        name : str
+            The name of the input drive
+        t0 : float
+            The start time (in ms).
+        T : float
+            The end time (in ms).
+        lamtha : float
+            The rate parameter for spike train (in Hz)
+        target_cell_type : str
+            'L2Basket', 'L2Pyr' etc.
+        weight_ampa : float
+            The weight of the AMPA connections.
+        weight_nmda : float
+            The weight of the NMDA connections.
+        spatial_lamtha : float
+            The decay
+        loc : str
+            'proximal' or 'distal'
```

Comment:
Do you envision calling `add_poisson_drive` once for each `target_cell_type`...? I guess that's the most explicit it can be, perhaps overly so though?